src/components/FilterBar/SavedSearchesList/SavedSearchesList.react.js

```diff
@@ -0,0 +1,57 @@
+import {SavedSearchesListItem} from './SavedSearchesListItem.react';
+
+export class SavedSearchesList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = this.getStateFromStores();
+    this.props.filterBarStore.addChangeListener(this._onChange.bind(this));
```

Comment:
Also it doesn't make sense that its listening for a change on the filterBarStore. As mentioned in that class the saved search stuff should be extracted into its own store.